File: Core/Entities/Work.cs
using System;

namespace Core.Entities
{
    public class Work : BaseEntity
    {

        public string Title { get; set; }
        public string Description { get; set; }
        public bool IsDone { get; set; } = false;
        public DateTime? GoalTime { get; set; }
        public User Owner { get; set; }
        public int OwnerId { get; set; }
        public User Person { get; set; }
        public int? PersonId { get; set; }
    }
}

File: client/src/app/shared/models/work.ts
export interface IWork {
    title: string;
    description: string;
    isDone: boolean;
    goalTime: string;
    id: number;
    personId: number;
    ownerId: number;
  }

export class Work {
    title: string;
    description: string;
    isDone: boolean;
    goalTime: string;
    id: number;
    personId: number;
    ownerId: number;
}
